decentralization.adoc

```diff
@@ -181,14 +180,14 @@ ____
 
 In the article, Van Wirdum interviews Bitcoin developers about their views on
 decentralization and the risks involved in increasing
-Bitcoin's maximum block size. This was a hot topic during the 2014-2017 era
+Bitcoin's maximum block size. This discussion was a hot topic during the 2014-2017 era
 when many people argued over increasing the block size limit to
 allow for more transaction throughput.
 
 A powerful argument against increasing the block size is that it
-increases the cost of verification (see <<_vertical_scaling,the
-Scaling chapter>>). If verification cost increases, it will push some
-full node users to stop running their full nodes. This in turn leads
+increases the cost of verification (see <<_vertical_scaling, the
```

Comment:
This is a cross-reference with a custom text. I prefer to not have the space in front of the custom text.